README.md

```diff
@@ -25,21 +56,44 @@ The unit tests for the API can be run by executing the following command:
 
     make test
 
+Note that the unit tests rely on the project having being started. (make start)
+
 ## Running the API
 
-The API can be run by running the following command:
+The API and the supporting components can be started by running the following command:
 
     make start
 
 When the API is running (it currently runs along HTTP on port 9095, but this will be changed to HTTPS shortly), the URL (home page) to the API is: http://127.0.0.1:9095/dosapi/dosservices/v0.0.1
 
 The API has a single GET Rest endpoint:
+
 - /byfuzzysearch
 
 The API will return a list of matching services in JSON format. This is defined on Confluence: https://nhsd-confluence.digital.nhs.uk/display/SFDEV/Known+Service+Search
```

Comment:
Can we format the link using `[Known Service Search](https://nhsd-confluence.digital.nhs.uk/display/SFDEV/Known+Service+Search)`

Reply:
Nice, done.